RecommendationEngine/owlbot.py

```diff
@@ -64,16 +59,6 @@
     r"\$transportConfig, and any \$serviceAddress",
     r"$transportConfig, and any `$apiEndpoint`")
 
-# fix year
-s.replace(
-    'src/**/**/*.php',
-    r'Copyright \d{4}',
-    r'Copyright 2020')
-s.replace(
-    'tests/**/**/*Test.php',
-    r'Copyright \d{4}',
-    r'Copyright 2020')
-
 # Change the wording for the deprecation warning.
 s.replace(
     'src/*/*_*.php',
```

Comment:
`RecommendationEngine` falls in the category of APIs which are not yet GA. We can add excludes for `*_*.php` and `class_alias`, and remove the block to replace the wording for the deprecation warning in `*_*.php`.

Reply:
Done